package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "designsystem",
-  "version": "1.2.77",
+  "version": "1.2.80",
```

Comment:
Remember to update verison in package.json of designsystem as well.

Reply:
You shouldn't bump the version number yourself unless you are making major/minor changes